ee/query-service/app/server.go

```diff
@@ -110,6 +111,10 @@ func NewServer(serverOptions *ServerOptions) (*Server, error) {
 
 	baseexplorer.InitWithDSN(baseconst.RELATIONAL_DATASOURCE_PATH)
 
+	if err := preferences.InitDB(baseconst.RELATIONAL_DATASOURCE_PATH); err != nil {
```

Comment:
The schema should be bootstrapped with dao as in line 107.

Is there any specific reason this is being bootstrapped separately?

Reply:
it helps keep the module initialization separate. like we do for dashboards and alerts in the follow up line